jgiven-gradle-plugin/build.gradle

```diff
@@ -5,21 +5,30 @@ plugins {
 }
 
 dependencies {
+    implementation localGroovy()
     implementation project(':jgiven-core')
     implementation project(':jgiven-html5-report')
     implementation "com.google.guava:guava:$guavaVersion"
     implementation "com.google.code.gson:gson:$gsonVersion"
 
-    testImplementation project(':jgiven-junit')
-    testImplementation 'org.spockframework:spock-core:1.3-groovy-2.5'
+    testImplementation project(':jgiven-junit5')
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.7.1'
+    testImplementation 'org.spockframework:spock-core:2.0-M4-groovy-3.0'
```

Comment:
Spock2 removes junit4 as a dependency. It will require `spock-junit4` dependency to make work JUnit4 Rules

Reply:
yes, I just want to migrate the tests we have to JUnit5. I have actually started doing so :) (although there are still a lot fo TODOs)

Reply:
But first I need to figure out how to make the plugin conform to Gradle 7

Reply:
I would rather leave Gradle Plugins to use Spock instead of JUnit5

Reply:
@jsalinaspolo  so you propose migrating the java/junit tests to groovy/spock?